python/pyspark/pandas/__init__.py

```diff
@@ -49,7 +49,8 @@
 ):
     import logging
 
-    logging.warning(
+    logger = logging.getLogger(__name__)
+    logger.warning(
```

Comment:
Let's use `warnings.warn` instead since PySpark doesn't leverage logging module for now.

Reply:
even better indeed